### ISSUE TITLE
How Can I get the folder "label_venacava" and "label_portalvein"

### ISSUE THREAD

**Issue by rw1995** [lemoshu/MTCL#3]
Hello, thank you for your open source code.

I have two questions to ask you.

1. I got the 3Dircadb-01 data set. How to generate the "label_venacava" folder and "label_portalvein" folder  in the corresponding code, because I did not see the protalvein folder in some patient folders?

2. Which groups in the 3Dircadb-01 data set did you use for training?

Can you help me?

**Comment by lemoshu** [comment]
Sry for replying late. I am very busy recently. 
1.I just downloaded the raw 3d ircadb dataset again and checked the original folder. For the "label_xx" folder, we manually constructed these folders. For the vessel labels, I found all the patients have protalvein masks? I guess you are confused about the venacava masks? Yes, some patients don’t have venacava consistently named folder, they named them venoussystem. Try to manually reconstruct them. 

2.We randomly split img 2,8,3,10,16,19,12,5,7,14 in ircadb as test set. The remaining samples and all MSD data are used for the training set.

3.other tips: empirically, careful preprocessing is far more important in this task. You can also visualize and see the intensity varies greatly among images whatever in IRCADb or MSD. Most of the semi-supervised method has a strong assumption that the data distribution should be very similar, domain shift appears obviously across images in both two datasets. that’s another challenge for this task, in fact, we adopted a “stupid” way that spent human effort for examination and manually rectified the HU ranges for some cases with highly different intensity ranges in our exploratory fully-supervised studies, along with concatenating the Sato-based prob map to tackle this. We ignored recording the specific “cases” under special treatments at that time. We plan to improved this method and extended it to journal version in the near future, which will not limit to this vessel seg task only. We will release the preprocessed data for this task after finishing our extended version. 

Thanks for your understanding and attention.

**Comment by rw1995** [comment]
> Sry for replying late. I am very busy recently.
> 1.I just downloaded the raw 3d ircadb dataset again and checked the original folder. For the "label_xx" folder, we manually constructed these folders. For the vessel labels, I found all the patients have protalvein masks? I guess you are confused about the venacava masks? Yes, some patients don’t have venacava consistently named folder, they named them venoussystem. Try to manually reconstruct them.
> 
> 2.We randomly split img 2,8,3,10,16,19,12,5,7,14 in ircadb as test set. The remaining samples and all MSD data are used for the training set.
> 
> 3.other tips: empirically, careful preprocessing is far more important in this task. You can also visualize and see the intensity varies greatly among images whatever in IRCADb or MSD. Most of the semi-supervised method has a strong assumption that the data distribution should be very similar, domain shift appears obviously across images in both two datasets. that’s another challenge for this task, in fact, we adopted a “stupid” way that spent human effort for examination and manually rectified the HU ranges for some cases with highly different intensity ranges in our exploratory fully-supervised studies, along with concatenating the Sato-based prob map to tackle this. We ignored recording the specific “cases” under special treatments at that time. We plan to improved this method and extended it to journal version in the near future, which will not limit to this vessel seg task only. We will release the preprocessed data for this task after finishing our extended version.
> 
> Thanks for your understanding and attention.

Thank you.  Looking forward to your further development
